catalog/spatial/spatial-commands/src/main/java/org/codice/ddf/commands/spatial/geonames/GazetteerUpdateCommand.java

```diff
@@ -31,15 +35,12 @@
 
 @Service
 @Command(
-  scope = "geonames",
+  scope = "gazetteer",
   name = "update",
-  description =
-      "Adds new entries to an existing local GeoNames index. "
-          + "Attempting to "
-          + "add entries when no index exists is an error."
+  description = "Updates the gazetter entries from a resource"
 )
-public final class GeoNamesUpdateCommand implements Action {
-  private static final Logger LOGGER = LoggerFactory.getLogger(GeoNamesUpdateCommand.class);
+public final class GazetteerUpdateCommand implements Action {
```

Comment:
Should `geonames` also be replaced from the package name?

Reply:
renamed